measure/src/macros.rs

```diff
@@ -6,26 +6,29 @@
 ///
 /// Use `measure_us!()` when you want to measure an expression in microseconds.
 ///
+/// Use `measdur!()` when you want to measure an expression and get the Duration.
+///
 /// [`Measure`]: crate::measure::Measure
 ///
 /// # Examples
 ///
 /// ```
 /// // Measure functions
-/// # use solana_measure::{measure_time, measure_us};
+/// # use solana_measure::{measure_time, measure_us, measdur};
```

Comment:
nit: I realize it is longer, but thoughts on using `measure_duration` or `measure_dur`. This would be consistent with the two existing macros that fully spell out `measure`

Reply:
> cargo-fmt within macros can sometimes be less than ideal IMO

Ya, I've seen some goofy stuff as well

> but I think this one is worthy of an exception given that ...

Wanna meet in the middle and do `meas_dur` ? One extra character from you had, but also keeps the underscore break for different words scheme

Reply:
🤝